pkg/controller/controller.go

```diff
@@ -130,7 +130,7 @@ type Controller struct {
 	lastErrorStamp    time.Time
```

Comment:
Please submit this commit as separate PR so we can backport if something else depends on this.